prime-router/docs/standard-operating-procedures/update-observation-mapping-table.md

```diff
@@ -120,13 +120,232 @@ options:
+      spreadsheet.getRange('B2').activate();
+      spreadsheet.getRange('B9').activate();
+      spreadsheet.getActiveRange().autoFillToNeighbor(SpreadsheetApp.AutoFillSeries.DEFAULT_SERIES);
+      spreadsheet.getRange('I9').activate();
+      spreadsheet.getActiveRange().autoFillToNeighbor(SpreadsheetApp.AutoFillSeries.DEFAULT_SERIES);
+      spreadsheet.getRange('J9').activate();
+      spreadsheet.getActiveRange().autoFillToNeighbor(SpreadsheetApp.AutoFillSeries.DEFAULT_SERIES);
+      spreadsheet.getRange('K9').activate();
+      spreadsheet.getActiveRange().autoFillToNeighbor(SpreadsheetApp.AutoFillSeries.DEFAULT_SERIES);
+      spreadsheet.getRange('L9').activate();
+      spreadsheet.getActiveRange().autoFillToNeighbor(SpreadsheetApp.AutoFillSeries.DEFAULT_SERIES);
+      spreadsheet.getRange('M9').activate();
+      spreadsheet.getActiveRange().autoFillToNeighbor(SpreadsheetApp.AutoFillSeries.DEFAULT_SERIES);
+      spreadsheet.getRange('A8:M8').activate();
+      spreadsheet.getActiveRangeList().setBackground('#cccccc');
+      spreadsheet.getRange('A9').activate();
+      spreadsheet.getActiveSheet().autoResizeColumns(1, 1);
+      spreadsheet.getActiveSheet().autoResizeColumns(2, 1);
+      spreadsheet.getActiveSheet().autoResizeColumns(6, 1);
+      spreadsheet.getActiveSheet().autoResizeColumns(7, 1);
+      spreadsheet.getActiveSheet().autoResizeColumns(11, 1);
+      spreadsheet.getActiveSheet().autoResizeColumns(13, 1);
+      spreadsheet.getRange('8:8').activate();
+      spreadsheet.getActiveSheet().setFrozenRows(8);
+      spreadsheet.getRange('B1').activate();
+      spreadsheet.getActiveRangeList().setBackground('#fff2cc');
+      spreadsheet.getRange('F1').activate();
+      spreadsheet.getCurrentCell().setRichTextValue(SpreadsheetApp.newRichTextValue()
+      .setText('Note: Change the sheet name to grab values from another sheet. Rows 1-7 are used to facilitate formulas, delete if exporting to update the observation mapping table.')
+      .setTextStyle(0, 99, SpreadsheetApp.newTextStyle()
+      .setFontFamily('Arial')
+      .build())
+      .build());
+      spreadsheet.getRange('F2').activate();
+    };
+    
+    ```
+    </details>
+
+- Export rows from all three sheets to CSV file and use it to replace `observation-mapping.csv`.
+- Add rows for AOE Questions:
+    - Either copy the values from the website and map them into the appropriate columns 
+      - [LOINC General](https://loinc.org/81959-9)
+      - [LOINC COVID](https://loinc.org/sars-cov-2-and-covid-19/#aoe)
+
+  -OR-  
+    - Sign up for an account, download the CSV, and map the data from it
+- Add rows for codes that have been added ad-hoc (not coming from any standardized source). 
+Look for these codes in the current version of `observation-mapping.csv` file. They are listed at the end
+and have `Condition Code System` = `ReportStream`.
+
```

Comment:
We should create a ticket to automate that. We could make it so that we have the option to pull LOCAL and ReportStream codes to the CSV file you're updating.